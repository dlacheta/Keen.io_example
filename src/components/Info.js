import React from 'react';

const Info = () => (
  <React.Fragment>
    <h2>Info</h2>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis non aliquam risus, eu tincidunt
      sapien. Etiam vel tempus purus. Morbi pharetra sem tortor, vitae imperdiet leo euismod quis.
      Maecenas enim risus, ultrices placerat laoreet ac, cursus consequat risus. Vivamus porttitor
      facilisis aliquam. Etiam sodales ut risus at mollis. Ut sit amet augue vitae tortor pretium
      fringilla a et sapien. Proin quis lobortis quam. Nullam semper mauris laoreet vestibulum
      tincidunt. Quisque elementum sed ligula in cursus. Vivamus ultricies turpis ut nisl bibendum
      tempor. In scelerisque neque vitae ante condimentum tristique. Maecenas dignissim suscipit
      tortor eget egestas. Aenean id nunc eget arcu imperdiet aliquet ac a sem. Quisque sodales
      tellus ac massa vehicula eleifend.
    </p>
    <p>
      Donec aliquet ex risus, et posuere purus feugiat a. Ut convallis justo sapien, a cursus dolor
      dictum at. Morbi porttitor tempus urna id commodo. Curabitur egestas velit purus, in bibendum
      magna efficitur nec. Sed tempor nulla eget lobortis aliquet. Cras ornare id mi eget
      consectetur. Integer cursus, enim ac placerat consectetur, ante elit efficitur lacus, vitae
      elementum elit leo ut lectus.
    </p>
  </React.Fragment>
);

export default Info;
