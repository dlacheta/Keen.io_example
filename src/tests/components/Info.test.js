import React from 'react';
import { shallow } from 'enzyme';
import Info from '../../components/Info';

it('Info component renders correctly', () => {
  const wrapper = shallow(<Info />);
  expect(wrapper).toMatchSnapshot();
});
