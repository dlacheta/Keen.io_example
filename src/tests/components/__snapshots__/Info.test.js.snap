// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Info component renders correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Info />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": Array [
        <h2>
          Info
        </h2>,
        <p>
          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis non aliquam risus, eu tincidunt sapien. Etiam vel tempus purus. Morbi pharetra sem tortor, vitae imperdiet leo euismod quis. Maecenas enim risus, ultrices placerat laoreet ac, cursus consequat risus. Vivamus porttitor facilisis aliquam. Etiam sodales ut risus at mollis. Ut sit amet augue vitae tortor pretium fringilla a et sapien. Proin quis lobortis quam. Nullam semper mauris laoreet vestibulum tincidunt. Quisque elementum sed ligula in cursus. Vivamus ultricies turpis ut nisl bibendum tempor. In scelerisque neque vitae ante condimentum tristique. Maecenas dignissim suscipit tortor eget egestas. Aenean id nunc eget arcu imperdiet aliquet ac a sem. Quisque sodales tellus ac massa vehicula eleifend.
        </p>,
        <p>
          Donec aliquet ex risus, et posuere purus feugiat a. Ut convallis justo sapien, a cursus dolor dictum at. Morbi porttitor tempus urna id commodo. Curabitur egestas velit purus, in bibendum magna efficitur nec. Sed tempor nulla eget lobortis aliquet. Cras ornare id mi eget consectetur. Integer cursus, enim ac placerat consectetur, ante elit efficitur lacus, vitae elementum elit leo ut lectus.
        </p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Info",
        },
        "ref": null,
        "rendered": "Info",
        "type": "h2",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis non aliquam risus, eu tincidunt sapien. Etiam vel tempus purus. Morbi pharetra sem tortor, vitae imperdiet leo euismod quis. Maecenas enim risus, ultrices placerat laoreet ac, cursus consequat risus. Vivamus porttitor facilisis aliquam. Etiam sodales ut risus at mollis. Ut sit amet augue vitae tortor pretium fringilla a et sapien. Proin quis lobortis quam. Nullam semper mauris laoreet vestibulum tincidunt. Quisque elementum sed ligula in cursus. Vivamus ultricies turpis ut nisl bibendum tempor. In scelerisque neque vitae ante condimentum tristique. Maecenas dignissim suscipit tortor eget egestas. Aenean id nunc eget arcu imperdiet aliquet ac a sem. Quisque sodales tellus ac massa vehicula eleifend.",
        },
        "ref": null,
        "rendered": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis non aliquam risus, eu tincidunt sapien. Etiam vel tempus purus. Morbi pharetra sem tortor, vitae imperdiet leo euismod quis. Maecenas enim risus, ultrices placerat laoreet ac, cursus consequat risus. Vivamus porttitor facilisis aliquam. Etiam sodales ut risus at mollis. Ut sit amet augue vitae tortor pretium fringilla a et sapien. Proin quis lobortis quam. Nullam semper mauris laoreet vestibulum tincidunt. Quisque elementum sed ligula in cursus. Vivamus ultricies turpis ut nisl bibendum tempor. In scelerisque neque vitae ante condimentum tristique. Maecenas dignissim suscipit tortor eget egestas. Aenean id nunc eget arcu imperdiet aliquet ac a sem. Quisque sodales tellus ac massa vehicula eleifend.",
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Donec aliquet ex risus, et posuere purus feugiat a. Ut convallis justo sapien, a cursus dolor dictum at. Morbi porttitor tempus urna id commodo. Curabitur egestas velit purus, in bibendum magna efficitur nec. Sed tempor nulla eget lobortis aliquet. Cras ornare id mi eget consectetur. Integer cursus, enim ac placerat consectetur, ante elit efficitur lacus, vitae elementum elit leo ut lectus.",
        },
        "ref": null,
        "rendered": "Donec aliquet ex risus, et posuere purus feugiat a. Ut convallis justo sapien, a cursus dolor dictum at. Morbi porttitor tempus urna id commodo. Curabitur egestas velit purus, in bibendum magna efficitur nec. Sed tempor nulla eget lobortis aliquet. Cras ornare id mi eget consectetur. Integer cursus, enim ac placerat consectetur, ante elit efficitur lacus, vitae elementum elit leo ut lectus.",
        "type": "p",
      },
    ],
    "type": Symbol(react.fragment),
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": Array [
          <h2>
            Info
          </h2>,
          <p>
            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis non aliquam risus, eu tincidunt sapien. Etiam vel tempus purus. Morbi pharetra sem tortor, vitae imperdiet leo euismod quis. Maecenas enim risus, ultrices placerat laoreet ac, cursus consequat risus. Vivamus porttitor facilisis aliquam. Etiam sodales ut risus at mollis. Ut sit amet augue vitae tortor pretium fringilla a et sapien. Proin quis lobortis quam. Nullam semper mauris laoreet vestibulum tincidunt. Quisque elementum sed ligula in cursus. Vivamus ultricies turpis ut nisl bibendum tempor. In scelerisque neque vitae ante condimentum tristique. Maecenas dignissim suscipit tortor eget egestas. Aenean id nunc eget arcu imperdiet aliquet ac a sem. Quisque sodales tellus ac massa vehicula eleifend.
          </p>,
          <p>
            Donec aliquet ex risus, et posuere purus feugiat a. Ut convallis justo sapien, a cursus dolor dictum at. Morbi porttitor tempus urna id commodo. Curabitur egestas velit purus, in bibendum magna efficitur nec. Sed tempor nulla eget lobortis aliquet. Cras ornare id mi eget consectetur. Integer cursus, enim ac placerat consectetur, ante elit efficitur lacus, vitae elementum elit leo ut lectus.
          </p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Info",
          },
          "ref": null,
          "rendered": "Info",
          "type": "h2",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis non aliquam risus, eu tincidunt sapien. Etiam vel tempus purus. Morbi pharetra sem tortor, vitae imperdiet leo euismod quis. Maecenas enim risus, ultrices placerat laoreet ac, cursus consequat risus. Vivamus porttitor facilisis aliquam. Etiam sodales ut risus at mollis. Ut sit amet augue vitae tortor pretium fringilla a et sapien. Proin quis lobortis quam. Nullam semper mauris laoreet vestibulum tincidunt. Quisque elementum sed ligula in cursus. Vivamus ultricies turpis ut nisl bibendum tempor. In scelerisque neque vitae ante condimentum tristique. Maecenas dignissim suscipit tortor eget egestas. Aenean id nunc eget arcu imperdiet aliquet ac a sem. Quisque sodales tellus ac massa vehicula eleifend.",
          },
          "ref": null,
          "rendered": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis non aliquam risus, eu tincidunt sapien. Etiam vel tempus purus. Morbi pharetra sem tortor, vitae imperdiet leo euismod quis. Maecenas enim risus, ultrices placerat laoreet ac, cursus consequat risus. Vivamus porttitor facilisis aliquam. Etiam sodales ut risus at mollis. Ut sit amet augue vitae tortor pretium fringilla a et sapien. Proin quis lobortis quam. Nullam semper mauris laoreet vestibulum tincidunt. Quisque elementum sed ligula in cursus. Vivamus ultricies turpis ut nisl bibendum tempor. In scelerisque neque vitae ante condimentum tristique. Maecenas dignissim suscipit tortor eget egestas. Aenean id nunc eget arcu imperdiet aliquet ac a sem. Quisque sodales tellus ac massa vehicula eleifend.",
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Donec aliquet ex risus, et posuere purus feugiat a. Ut convallis justo sapien, a cursus dolor dictum at. Morbi porttitor tempus urna id commodo. Curabitur egestas velit purus, in bibendum magna efficitur nec. Sed tempor nulla eget lobortis aliquet. Cras ornare id mi eget consectetur. Integer cursus, enim ac placerat consectetur, ante elit efficitur lacus, vitae elementum elit leo ut lectus.",
          },
          "ref": null,
          "rendered": "Donec aliquet ex risus, et posuere purus feugiat a. Ut convallis justo sapien, a cursus dolor dictum at. Morbi porttitor tempus urna id commodo. Curabitur egestas velit purus, in bibendum magna efficitur nec. Sed tempor nulla eget lobortis aliquet. Cras ornare id mi eget consectetur. Integer cursus, enim ac placerat consectetur, ante elit efficitur lacus, vitae elementum elit leo ut lectus.",
          "type": "p",
        },
      ],
      "type": Symbol(react.fragment),
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
